library/core/src/bool.rs

```diff
@@ -2,7 +2,7 @@
 
 #[lang = "bool"]
 impl bool {
-    /// Returns `Some(t)` if the `bool` is [`true`](keyword.true.html), or `None` otherwise.
+    /// Returns `Some(t)` if the `bool` is `true`, or `None` otherwise.
```

Comment:
This is not right, you should use relative links rather than removing the link altogether.